ckanext/dcat/profiles/base.py

```diff
@@ -465,7 +465,8 @@ def _contact_details(self, subject, predicate):
 
         for agent in self.g.objects(subject, predicate):
 
-            contact["uri"] = str(agent) if isinstance(agent, term.URIRef) else ""
+            contact["uri"] = (str(agent) if isinstance(agent, URIRef)
+                        else self._get_vcard_property_value(agent, VCARD.hasUID))
```

Comment:
Let's keep this as it was and not assign hasUID to `uri` as discussed. 
Otherwise this is good to go

Reply:
I reverted this part back and added also the new mapping to the documentation